source/features/expand-all-collapsed-code.tsx

```diff
@@ -0,0 +1,77 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
```

Comment:
This this selector is used multiple times in the file, you can define it here.

```suggestion

const expanderSelector = '.js-expand.directional-expander.single-expander';
```